### ISSUE TITLE
Accordion/Collapse panel markup missing `role="region"`

### ISSUE THREAD

**Issue by c-vetter** [htmlstreamofficial/preline#373]
The accordion and collapse panels are presented with markup like this:
```html
<div id="hs-basic-collapse-three" class="hs-accordion-content hidden w-full overflow-hidden transition-[height] duration-300" aria-labelledby="hs-basic-heading-three">
```

This way, `aria-labelledby` is ignored. To make it work correctly, add `role="region"`, like so:
```diff
- <div id="hs-basic-collapse-three" class="hs-accordion-content hidden w-full overflow-hidden transition-[height] duration-300" aria-labelledby="hs-basic-heading-three">
+ <div id="hs-basic-collapse-three" class="hs-accordion-content hidden w-full overflow-hidden transition-[height] duration-300" aria-labelledby="hs-basic-heading-three" role="region">
```

See https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/

**Comment by jahaganiev** [comment]
Hey @c-vetter - thanks for the suggestions. We will re-visit aria data attributes in future updates. Thanks!

Duplicated #365

**Comment by jahaganiev** [comment]
Hey @c-vetter - our team released v2.4.0 update which includes enhancements for this case. Thanks!
